scripts/process_output.py

```diff
@@ -307,29 +307,38 @@ def process_metadata(ds):
         The path of a new directory where all data should be stored for this run.
     """
     s = json.dumps(ds.attrs, cls=NumpyEncoder)
+    metadata = json.loads(s) 
     # Hash based on the metadata of the run
-    run_id = hashlib.sha256(s.encode("utf-8")).hexdigest()[:8]
+    hash = hashlib.sha256(s.encode("utf-8")).hexdigest()[:8]    
+    metadata["hash"] = hash
+    # Determine the institute based on the project name
+    project = metadata.get("project", "")
+    if project.startswith("a8b1"):
+        institute = "SWPC"
+    elif project.startswith("ENLIL."):
+        institute = "CCMC"
+    elif project.startswith("/data"):
+        institute = "SWxTREC"
+    else:
+        institute = "GMU"
+        warnings.warn("Unknown institute, defaulting to GMU")
+    metadata["institute"] = institute
+
+    # Get the run_id from the filename if the institute is SWPC, otherwise set it to the hash
+    if institute == "SWPC":
+        run_id = path.name.split("_")[-1].split(".")[0]
+        if not run_id:
+            raise ValueError("SWPC run metadata does not contain a run_id")
```

Comment:
Is this possible to hit? I think your splits and indexing above would error before getting here.